terraform/modules/paas/main.tf

```diff
@@ -46,6 +46,19 @@ resource cloudfoundry_service_instance postgres_snapshot {
   count        = var.snapshot_databases_to_deploy
   name         = local.postgres_snapshot_service_name
   space        = data.cloudfoundry_space.space.id
+  service_plan = data.cloudfoundry_service.postgres.service_plans[var.postgres_snapshot_service_plan_13]
+  json_params  = jsonencode(local.postgres_params)
+  timeouts {
+    create = "30m"
+    delete = "30m"
+    update = "30m"
+  }
+}
+
+resource cloudfoundry_service_instance postgres_snapshot_13 {
+  count        = var.snapshot_databases_to_deploy
+  name         = local.postgres_snapshot_service_name_13
+  space        = data.cloudfoundry_space.space.id
   service_plan = data.cloudfoundry_service.postgres.service_plans[var.postgres_snapshot_service_plan]
```

Comment:
should this be var.postgres_snapshot_service_plan_13?

Reply:
Indeed, thanks. I also forgot a bunch of stuff... should be fixed now